CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Fixes
 
 * Fix pre tag parsing for `partition_html`
+* Fix lookup error for annotated Arabic and Hebrew encodings
```

Comment:
also would be nice to bump the version to `-dev2` here (and `usntructured/__version__.py`). though tbh, sometimes that gets overlooked (esp. if another change merges ahead of yours to the same version)

Reply:
done! so for future reference, every PR should update the changelog and bump the number on the dev version?

Reply:
ideally, yes, if there are code changes